client/landing/gutenboarding/hooks/use-on-site-creation.ts

```diff
@@ -139,11 +149,7 @@ export default function useOnSiteCreation() {
 				return;
```

Comment:
Just noting that this would be another place to add in the call, but since `'gutenboarding/plans-grid'` is enabled everywhere, this block isn't currently used. I imagine we'll be removing this block anyway, so it probably doesn't need to be added here, but thought I'd mention it just in case!